src/components/back-link/template.test.js

```diff
@@ -0,0 +1,74 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const COMPONENT_NAME = 'back-link'
```

Comment:
Not 100% sure this is worth the comprehension cost - seems like an extra connection for contributors to make?

Reply:
Worth noting, we also specify the class name in every test and it feels like if we continue down this path we would make that a constant too…?

e.g.

```
const { $ } = render(COMPONENT_NAME, examples.default)

const $component = $(COMPONENT_CLASS)
```

which I think could be a bit intimidating for new contributors.

Reply:
Agree let's remove this constant